doc/installing.md

```diff
@@ -60,4 +60,10 @@ $ make install
 ```
 The default installation prefix is `/usr/local`. Change this to your liking.
 
+Other cmake variables that maybe useful to set are
+* `BOOST_ROOT` path to the root directory of Boost
+* `CMAKE_C_COMPILER` C compiler to use
```

Comment:
I agree with @fodinabor that redocumenting `CMAKE_C_COMPILER` is definitely unnecessary since hipSYCL does not use `CMAKE_C_COMPILER` anywhere - adding it might maybe rather confuse people. Can you remove it?

Reply:
Done.